ci/validate_wheel.sh

```diff
@@ -7,6 +7,25 @@ package_dir=$1
 wheel_dir_relative_path=$2
 
 
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+
+# some packages are much larger on CUDA 11 than on CUDA 12
+if [[ "${package_dir}" == "python/libcuvs" ]]; then
+    if [[ "${RAPIDS_CUDA_MAJOR}" == "11" ]]; then
+        PYDISTCHECK_ARGS=(
+            --max-allowed-size-compressed '1.0G'
+        )
+    else
+        PYDISTCHECK_ARGS=(
+            --max-allowed-size-compressed '875M'
+        )
+    fi
+else
+    PYDISTCHECK_ARGS=()
+fi
```

Comment:
You'll need to also add this to the `pydistcheck` call a few lines down, like:

```shell
pydistcheck \
    --inspect \
    "${PYDISTCHECK_ARGS[@]}" \
    "$(echo "${wheel_dir_relative_path}"/*.whl)"
``

Reply:
yep, flying fast and loose here

Reply:
to quote School of Rock: "you're not hardcore unless you live hardcore" 🤘🏻